pyodide_build/recipe/builder.py

```diff
@@ -523,8 +529,7 @@ def _build_package(self, bash_runner: BashRunnerWithSharedEnvironment) -> None:
             self._compile(bash_runner)
 
         self._package_wheel(bash_runner)
-        shutil.rmtree(self.dist_dir, ignore_errors=True)
-        shutil.copytree(self.src_dist_dir, self.dist_dir)
+        shutil.copytree(self.src_dist_dir, self.dist_dir, dirs_exist_ok=True)
```

Comment:
Leave the old wheels instead of removing them. They can be reused on a branches where they still have the appropriate package version, python version, and abi version.